test/membrane_mp4/muxer/cmaf/cmaf_test.exs

```diff
@@ -1,52 +0,0 @@
-defmodule Membrane.MP4.Muxer.CMAF.Segment.HelperTest do
```

Comment:
shouldn't we put that back?

Reply:
It tests a function that no longer exists so I'm not sure about that.

Well, I'll probably create a different test checking what happens in case of new stream parameters inside of cmaf instead of this one.